src/datomish/query/source.cljc

```diff
@@ -35,13 +35,17 @@
 (defprotocol Source
   (source->from [source attribute]
     "Returns a pair, `[table alias]` for a pattern with the provided attribute.")
+  (source->non-fts-from [source])
```

Comment:
Would you mind always using `fulltext` in names?  Might as well make it easy to grep.